addon/instance-initializers/hot-loader-livereload-plugin.js

```diff
@@ -15,22 +36,10 @@ function createPlugin (appName, hotReloadService) {
       // Reloading app.js will fire Application.create unless we set this.
       // TODO: make sure this doesn't break tests
       window.runningTests = true;
-      var tags = document.getElementsByTagName('script');
-      for (var i = tags.length; i >= 0; i--){
-        if (tags[i] && tags[i].getAttribute('src') != null && tags[i].getAttribute('src').indexOf(appName) !== -1) {
-          tags[i].parentNode.removeChild(tags[i]);
-        }
-      }
-      var script = document.createElement('script');
-      script.onload = function() {
-        setTimeout(function() {
-          window.runningTests = false;
-          hotReloadService.trigger('willHotReload', path);
-        }, 10);
-      };
-      script.type = 'text/javascript';
-      script.src = `/assets/${appName}.js`;
-      document.body.appendChild(script);
+      all([reloadScript(`/assets/${appName}.js`), reloadScript('/assets/vendor.js')]).then(()=>{
```

Comment:
very nice! thanks for continuing to improve upon my original setup here :) you're a 10x dude!!!

Reply:
Just as an FYI you will also need to think of babel polyfill as that throws an error when trying to load twice

Reply:
@toranb FYI, I see there is an issue to add support for addons, this will be needed for that to work. Unfortuantely, I won't have a lot of time to work on this directly, but I think the key here would be to split vendor.js into addon tree and vendor. There are multiple trees that all get merged at the end, maybe we could remove the one for loader.js and the other problematic libraries.